src/test/java/com/wikia/webdriver/testcases/mercurytests/NavigationTest.java

```diff
@@ -31,7 +31,7 @@ private void init() {
   }
 
   @Test(groups = "mercury-navigation-001")
```

Comment:
consider renaming groups to something more descriptive so when displayed on dashboard or in Jenkins there is more context available to the reader